README.Rmd

```diff
@@ -106,11 +106,9 @@ ccao::vars_dict %>%
     )
   ) %>%
   mutate(`Unique to Condo Model` = ifelse(
-    var_name_model != "loc_tax_municipality_name" & (
```

Comment:
This conditional is no longer necessary since the res model and the condo model should now be aligned on their use of `loc_cook_municipality_name`